packages/@aws-cdk/aws-redshift-alpha/lib/private/database-query-provider/privileges.ts

```diff
@@ -85,8 +103,21 @@ async function updatePrivileges(
     return tableAdded || actionsAdded;
   });
   if (tablesToGrant.length > 0) {
-    await grantPrivileges(username, tablesToGrant, clusterProps);
+    await grantPrivileges(username, tablesToGrant, clusterProps, stackId);
   }
 
   return { replace: false };
 }
+
+/**
+ * We need this normalization logic because some of the `TableName` values
+ * are physical IDs generated in the `./util.ts` module.
```

Comment:
I don't see a `util.ts` file in this module so I'm not sure how the TableName was generated. Can you share the link to the function?